paired-fastq-to-unmapped-bam.wdl

```diff
@@ -26,31 +29,46 @@
 
 # WORKFLOW DEFINITION
 workflow ConvertPairedFastQsToUnmappedBamWf {
-  Array[String] readgroup_list
-  Map[String, Array[File]] fastq_pairs
-  Map[String, Array[String]] metadata
+  File readgroup_list  
+  Array[Array[String]] readgroup_array = read_tsv(readgroup_list) 
+  String ubam_list_name = basename(readgroup_list,".list")
+  
+  String docker
+  Int? preemptible_attempts
 
   # Convert multiple pairs of input fastqs in parallel
-  scatter (readgroup in readgroup_list) {
+  scatter (i in range(length(readgroup_array))) {
```

Comment:
We switched to using a readgroup list file instead of an array because there isn't an option for map variables in FC for the other input variables (fastq_pairs and metadata)